libs/shared/db/mysql/account/src/lib/cart.ts

```diff
@@ -57,16 +58,28 @@ export class Cart extends BaseModel {
 
   static async patchById(id: string, items: Partial<Cart>) {
```

Comment:
If we assume the typical use-case of a cart, is that someone fetches it, updates the instance, then wants to save it, these shouldn't be static methods, they should be normal methods.